src/stores/controller/controllerLogin.ts

```diff
@@ -12,9 +12,13 @@ import { getControllerRoutePrefix } from '@/lib/router'
 import { useThemeStore } from '../theme'
 
 export const useLoginStore = defineStore('controllerLogin', () => {
+  const TOKEN_REFRESH_INTERVAL = 1000 * 60 * 30 // half an hour
```

Comment:
This is bugging me like the one in [`src/stores/controller/units.ts`](https://github.com/NethServer/nethsecurity-ui/pull/243/files#diff-464476155c61f03f78cf041cbcc1df9a28a98d037290e8f95b9024d53a13be97R50)